OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ChildController.cs

```diff
@@ -103,11 +104,11 @@ public async Task<IActionResult> GetByParentIdForAdmin(Guid parentId, [FromQuery
     [ProducesResponseType(StatusCodes.Status403Forbidden)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [HttpGet]
-    public async Task<IActionResult> GetUsersChildren([FromQuery] OffsetFilter offsetFilter)
+    public async Task<IActionResult> GetUsersChildren([FromQuery] bool? isGetParent, [FromQuery] OffsetFilter offsetFilter)
     {
```

Comment:
We can use this optional parameter in controller like: 
public async Task<IActionResult> GetUsersChildren([FromQuery] OffsetFilter offsetFilter, [FromQuery] bool isGetParent = false)